dcapp/app/src/main/java/com/example/michael/mqtttest/MQTTConnectionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.example.michael.mqtttest;
+
+import org.eclipse.paho.client.mqttv3.IMqttActionListener;
+import org.eclipse.paho.client.mqttv3.IMqttToken;
+
+/**
+ * @desc - This is the callback class used to handle connection events from a
+ *         MQTT client. It's current just a stub but it could provide some
+ *         functionality if we need it later. (A call back class is needed
+ *         by MQTTclient to establish a connection).
+ */
+public class MQTTConnectionHandler implements IMqttActionListener {
+    @Override
+    public void onSuccess(IMqttToken iMqttToken) {
+        BroadcastService.connectionStatus = "Connected";
+    }
+
+    @Override
+    public void onFailure(IMqttToken iMqttToken, Throwable throwable) {
+        BroadcastService.connectionStatus = "Disconnected";
```

Comment:
Probably should make `"Connected"` and `"Disconnected"` into a `public static final` constant as they are used in other files as well.